.golangci-lint.yml

```diff
@@ -1,36 +1,28 @@
-# options for analysis running
-run:
-  concurrency: 1
-  deadline: 5m
-
-  # include test files or not
-  tests: true
-
-# output configuration options
-output:
-  # colored-line-number|line-number|json|tab|checkstyle
-  format: colored-line-number
-
 linters:
-  enable:
-    - errcheck
-    - unused
-    - dupl
-    - gosimple
-    - goconst
-    - maligned
-    - misspell
-    - unparam
-    - nakedret
-    - golint
-    - megacheck
-    - deadcode
-    - varcheck
-    - structcheck
-    - unconvert
-    - ineffassign
-    - gofmt
-    - gosec
-    - interfacer
-    - govet
-    - gosimple
+  enable-all: true
+  disable:
```

Comment:
Well, basically I enabled all and just disable some of them to not produce big changes in this PR (cause the pipeline would fail if we don't). I'll re-enable the ones with `FIXME` comments in the next PR(s).